mvn/archetypes/radix/src/main/resources/archetype-resources/solr/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM solr:5-alpine
```

Comment:
Clarification: Entrypoints will be inherited from base image here?

Reply:
yes @IMS94, the contents were copied into the $SOLR_HOME, and the base image is configured to load the conf files from there. I've tested it and it works fine.